packages/paypal-js/src/load-script.ts

```diff
@@ -104,6 +110,7 @@ function validateArguments(options: unknown, PromisePonyfill?: unknown) {
     if (typeof options !== "object" || options === null) {
         throw new Error("Expected an options object.");
     }
+
```

Comment:
should the `options.clientId` check come after line 112? I just noticed that `throw new Error("Expected an options object.");` will never be hit and/or the error message might be something like "cannot read attribute `clientId` of undefined"

Reply:
ah you're right! pls ignore my comment as 112 will throw before the client id check